dataframe-jdbc/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/h2Test.kt

```diff
@@ -0,0 +1,428 @@
+                    booleanCol BOOLEAN,
+                    tinyIntCol TINYINT,
+                    smallIntCol SMALLINT,
+                    integerCol INT,
+                    bigIntCol BIGINT,
+                    numericCol NUMERIC(10, 2),
+                    realCol REAL,
+                    doublePrecisionCol DOUBLE PRECISION,
+                    decFloatCol DECFLOAT(16),
+                    dateCol DATE,
+                    timeCol TIME,
+                    timeWithTimeZoneCol TIME WITH TIME ZONE,
+                    timestampCol TIMESTAMP,
+                    timestampWithTimeZoneCol TIMESTAMP WITH TIME ZONE,
+                    javaObjectCol OBJECT,
+                    enumCol VARCHAR(10),
+                    jsonCol JSON,
+                    uuidCol UUID
+                )
+            """.trimIndent()
+        )
+
+        connection.prepareStatement(
+            """
+                INSERT INTO TestTable VALUES (
+                    'ABC', 'XYZ', 'Long text data for CLOB', 'Medium text data for CLOB',
+                    'Varchar IgnoreCase', X'010203', X'040506', X'070809',
+                    TRUE, 1, 100, 1000, 100000,
+                    123.45, 1.23, 3.14, 2.71,
+                    '2023-07-20', '08:30:00', '18:15:00', '2023-07-19 12:45:30',
+                    '2023-07-18 12:45:30', NULL,
+                    'Option1', '{"key": "value"}', '123e4567-e89b-12d3-a456-426655440000'
+                )
+            """.trimIndent()
+        ).executeUpdate()
+
+        connection.prepareStatement(
+            """
+                INSERT INTO TestTable VALUES (
+                    'DEF', 'LMN', 'Another CLOB data', 'Different CLOB data',
+                    'Another Varchar', X'101112', X'131415', X'161718',
+                    FALSE, 2, 200, 2000, 200000,
+                    234.56, 2.34, 4.56, 3.14,
+                    '2023-07-21', '14:30:00', '22:45:00', '2023-07-20 18:15:30',
+                    '2023-07-19 18:15:30', NULL,
+                    'Option2', '{"key": "another_value"}', '234e5678-e89b-12d3-a456-426655440001'
+                )
+            """.trimIndent()
+        ).executeUpdate()
+
+        connection.prepareStatement(
+            """
+                INSERT INTO TestTable VALUES (
+                    'GHI', 'OPQ', 'Third CLOB entry', 'Yet another CLOB data',
+                    'Yet Another Varchar', X'192021', X'222324', X'252627',
+                    TRUE, 3, 300, 3000, 300000,
+                    345.67, 3.45, 5.67, 4.71,
+                    '2023-07-22', '20:45:00', '03:30:00', '2023-07-21 23:45:15',
+                    '2023-07-20 23:45:15', NULL,
+                    'Option3', '{"key": "yet_another_value"}', '345e6789-e89b-12d3-a456-426655440002'
+                )
+            """.trimIndent()
+        ).executeUpdate()
+
+        val df = DataFrame.readSqlTable(connection, "TestTable").cast<TestTableData>()
+        assertEquals(3, df.rowsCount())
```

Comment:
We usually use `shouldBe` from https://kotest.io/docs/assertions/assertions.html across the library

Reply:
I am not a big fan of that approach, but it should be consistent, thanks